### DISCUSSION TITLE
NGROK tunnel

### DISCUSSION THREAD

**Discussion by noahra** [miguelgrinberg/Flask-SocketIO#1988] [General]
Hi! I followed Miguel's guide on Flask-SocketIO and everything worked perfectly. However, I am unable to tunnel the app to NGROK. I need to make my app publicly available for a short while. I wrote a detailed question here at Stackoverflow:

https://stackoverflow.com/questions/76234832/mixed-content-the-page-at-this-request-has-been-blocked-this-endpoint-mus

**Comment by miguelgrinberg** [comment]
Have you tried using https and wss? That's the proper way to use Ngrok, at least that's what I always do. I don't think it makes sense to downgrade the security of the tunnel by disabling encryption

**Reply by noahra** [reply]
Thanks for replying, Miguel.

I tried this command

`$ ngrok http 5000`

Which by default is https. The issue is that the app then does not work. I get this error when I inspect in the browser:
<img width="1608" alt="Screenshot 2023-05-12 at 11 47 25" src="https://github.com/miguelgrinberg/Flask-SocketIO/assets/77496609/00a77262-3290-44fe-abb6-cfb473f476e2">

**Reply by miguelgrinberg** [reply]
Yes. You also have to change `ws://` to `wss://`.

**Reply by noahra** [reply]
Thank you so much, Miguel! My app works now!!

**Reply by rohit0407rai** [reply]
How did you solved it pls let me know...?
